back/spec/models/phase_spec.rb

```diff
@@ -45,6 +45,26 @@
       p = create(:phase, participation_method: 'budgeting')
       expect(p.save).to be true
     end
+
+    it 'can be changed from a transitive method to another one' do
+      phase = create :phase, participation_method: 'ideation'
+      phase.participation_method = 'budgeting'
+      expect(phase.save).to be true
+    end
+
+    it 'cannot be changed from a transitive method to a non-transitive one' do
```

Comment:
The term `non-transitive` is more general than what the test describes, because today we have only one. The test will have to be adapted when we add a new non-transitive method. But this is not blocking. It is just an observation.